extensions/amp-analytics/0.1/vendors.js

```diff
@@ -499,6 +497,94 @@ export const ANALYTICS_CONFIG = /** @type {!JSONType} */ ({
+          'id=${pixelId}' +
+          '&cd[content_name]=${content_name}',
+      'eventViewContent': '${base}&ev=ViewContent&' +
+          'id=${pixelId}' +
+          '&cd[value]=${value}' +
+          '&cd[currency]=${currency}' +
+          '&cd[content_name]=${content_name}' +
+          '&cd[content_type]=${content_type}' +
+          '&cd[content_ids]=${content_ids}',
+      'eventSearch': '${base}&ev=Search&' +
+          'id=${pixelId}' +
+          '&cd[value]=${value}' +
+          '&cd[currency]=${currency}' +
+          '&cd[content_category]=${content_category}' +
+          '&cd[content_ids]=${content_ids}' +
+          '&cd[search_string]=${search_string}',
+      'eventAddToCart': '${base}&ev=AddToCart&' +
+          'id=${pixelId}' +
+          '&cd[value]=${value}' +
+          '&cd[currency]=${currency}' +
+          '&cd[content_name]=${content_name}' +
+          '&cd[content_type]=${content_type}' +
+          '&cd[content_ids]=${content_ids}',
+      'eventAddToWishlist': '${base}&ev=AddToWishlist&' +
+          'id=${pixelId}' +
+          '&cd[value]=${value}' +
+          '&cd[currency]=${currency}' +
+          '&cd[content_name]=${content_name}' +
+          '&cd[content_category]=${content_category}' +
+          '&cd[content_ids]=${content_ids}',
+      'eventInitiateCheckout': '${base}&ev=InitiateCheckout&' +
+          'id=${pixelId}' +
+          '&cd[value]=${value}' +
+          '&cd[currency]=${currency}' +
+          '&cd[content_name]=${content_name}' +
+          '&cd[content_category]=${content_category}' +
+          '&cd[num_items]=${num_items}' +
+          '&cd[content_ids]=${content_ids}',
+      'eventAddPaymentInfo': '${base}&ev=AddPaymentInfo&' +
+          'id=${pixelId}' +
+          '&cd[value]=${value}' +
+          '&cd[currency]=${currency}' +
+          '&cd[content_category]=${content_category}' +
+          '&cd[content_ids]=${content_ids}',
+      'eventPurchase': '${base}&ev=Purchase&' +
+          'id=${pixelId}' +
+          '&cd[value]=${value}' +
+          '&cd[currency]=${currency}' +
+          '&cd[content_name]=${content_name}' +
+          '&cd[content_type]=${content_type}' +
+          '&cd[content_ids]=${content_ids}' +
+          '&cd[num_items]=${num_items}',
+      'eventLead': '${base}&ev=Lead&' +
+          'id=${pixelId}' +
+          '&cd[value]=${value}' +
+          '&cd[currency]=${currency}' +
+          '&cd[content_name]=${content_name}' +
+          '&cd[content_category]=${content_category}',
+      'eventCompleteRegistration': '${base}&ev=CompleteRegistration&' +
+          'id=${pixelId}' +
+          '&cd[value]=${value}' +
+          '&cd[currency]=${currency}' +
+          '&cd[content_name]=${content_name}' +
+          '&cd[status]=${status}',
+    },
+    'triggers': {
+      'trackPageview': {
+        'on': 'visible',
+        'request': 'pageview',
+      },
+    },
+    'transport': {
+      'beacon': true,
```

Comment:
I see. We would recommend turn `beacon` and `xhrpost` on if server support it. But if you only want pixel image to track Analytics for some reason, it it totally up to you. Please refer to [transport doc](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/amp-analytics.md#transport) for more information.
Please also note it can be overridden in user config as well.